src/expr/src/relation/mod.rs

```diff
@@ -1739,11 +1740,42 @@ impl MirRelationExpr {
         false
     }
 
+    /// Given the ids and values of a LetRec, it computes the subset of ids that are used across
+    /// iterations.
```

Comment:
Can you expand this comment with an example. For example, IIUC in

```sql
WITH MUTUALLY RECURSIVE
    x(...) AS f(z),
    y(...) AS g(x),
    z(...) AS h(y)
...;
```

the only recursive `LocalId` is `z` because `x` and `y` only occur in places that reference the value from the current iteration.